### DISCUSSION TITLE
question for recognizing data.xxx file

### DISCUSSION THREAD

**Discussion by Luckaswww** [deepmodeling/dpgen#1685] [Q&A]
Hello. I had 30 data.xxx files in the 02.fp folder and only recognized one data.xxx in the next iteration of training. Why is this? How can we fix it. 
deepmd-kit==2.2.6 dpgen=0.12.1 dpdata==0.2.19,lammps==LAMMPS (2 Aug 2023 - Update 1) 
![image](https://github.com/user-attachments/assets/f6a3b3c7-bded-4282-8b65-4560792ce46b)
![image](https://github.com/user-attachments/assets/a22ea961-85c1-46c3-a1b5-e48f2069f27c)

**Comment by njzjz** [comment]
Are you sure these directories are not empty?
